readthedocs/search/faceted_search.py

```diff
@@ -3,18 +3,21 @@
 from elasticsearch_dsl import FacetedSearch, TermsFacet
 from elasticsearch_dsl.query import Bool, SimpleQueryString
 
+from readthedocs.core.utils.extend import SettingsOverrideObject
 from readthedocs.search.documents import PageDocument, ProjectDocument
-from readthedocs.search.signals import before_file_search, before_project_search
 
 log = logging.getLogger(__name__)
 
 
 class RTDFacetedSearch(FacetedSearch):
```

Comment:
This class if it's not used here, should probably lives on corporate repo.

Reply:
Eh? It's definitely used on the .org.

Reply:
Fixed the comment.

Reply:
Nevermind. Got confused by the comment.